source/Octopus.Tentacle.Tests.Integration/ClientScriptExecutionScriptServiceV1IsNotRetried.cs

```diff
@@ -21,12 +21,10 @@ namespace Octopus.Tentacle.Tests.Integration
     public class ClientScriptExecutionScriptServiceV1IsNotRetried : IntegrationTest
     {
         [Test]
-        [TestCaseSource(typeof(TentacleTypesToTest))]
-        public async Task WhenANetworkFailureOccurs_DuringStartScript_WithATentacleThatOnlySupportsV1ScriptService_TheCallIsNotRetried(TentacleType tentacleType, SyncOrAsyncHalibut syncOrAsyncHalibut)
+        [TentacleConfigurations(testNoCapabilitiesServiceVersions: true)]
```

Comment:
`TentacleTypesToTest` is just the regular polling/listening/sync/async mix with no version.

But here, we've chosen `testNoCapabilitiesServiceVersions: true`, which now changes the versions.

What was the reason for that?

Whatever we choose, it's throughout this file.

Reply:
Yeah this is intentional.

All these tests used to explicitly call `.WithTentacleVersion(TentacleVersions.v6_3_451_NoCapabilitiesService)`, but now that is handled by the attribute bool and the test case `CreateBuilder` method.